base-cert-manager/certificates.yaml

```diff
@@ -0,0 +1,56 @@
+# https://cert-manager.io/docs/configuration/selfsigned/#bootstrapping-ca-issuers
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned-issuer
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: cockroachdb-ca
+spec:
+  isCA: true
+  commonName: cockroachdb-ca
+  dnsNames:
+    - cockroachdb-ca
+  secretName: cockroachdb-ca
+  issuerRef:
+    name: selfsigned-issuer
+    kind: Issuer
+    group: cert-manager.io
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: cockroachdb-ca-issuer
+spec:
+  ca:
+    secretName: cockroachdb-ca
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: node
+spec:
+  issuerRef:
+    name: cockroachdb-ca-issuer
+    kind: Issuer
+  commonName: node
+  dnsNames:
+    - cockroachdb-0.cockroachdb
+    - cockroachdb-1.cockroachdb
+    - cockroachdb-2.cockroachdb
```

Comment:
if someone adds more replicas to the STS will they need to update this too?

Reply:
> Are you suggesting to improve this or perhaps add a note somewhere to make it clear?

_if_ changing the number of replicas is something users are likely to do, then I think it might be worth noting somewhere the pieces that need to change. (I can't think of a way to improve it otherwise)